sdk/storage/Azure.Storage.Shared/src/Azure.Storage.Shared.shproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
As we chatted the dotnet cli doesn't support this project type cross-platform, it only works on Windows with a VS install. So I'd suggest just sharing the source files by including them directly in each project.

Reply:
I've removed these and replaced them with \Shared folders as we discussed.